web/src/features/map/Map.tsx

```diff
@@ -349,11 +364,64 @@ export default function MapPage(): ReactElement {
       mapStyle={MAP_STYLE as mapboxgl.Style}
     >
       <Layer id="ocean" type="background" paint={styles.ocean} />
-      <Source id="zones-clickable" promoteId={'zoneId'} type="geojson" data={geometries}>
+
+      <Source
+        id="zones-clickable"
+        promoteId={'zoneId'}
+        type="geojson"
+        data={worldGeometries}
+      >
         <Layer id="zones-clickable-layer" type="fill" paint={styles.zonesClickable} />
         <Layer id="zones-hoverable-layer" type="fill" paint={styles.zonesHover} />
         <Layer id="zones-border" type="line" paint={styles.zonesBorder} />
       </Source>
+      <Source id="states" type="geojson" data={statesGeometries}>
+        <Layer id="states-border" type="line" paint={styles.statesBorder} minzoom={2.5} />
+        <Layer
+          id="state-labels-name"
+          type="symbol"
+          source="states"
+          layout={{
+            'text-field': ['get', 'stateName'],
+            'symbol-placement': 'point',
+            'text-size': 12,
+            'text-letter-spacing': 0.12,
+            'text-transform': 'uppercase',
+
+            'text-font': ['Poppins SemiBold'],
+          }}
+          paint={{
+            'text-color': 'white',
+            'text-halo-color': '#111827',
+            'text-halo-width': 0.5,
+            'text-halo-blur': 0.25,
+            'text-opacity': 0.9,
```

Comment:
And same here with the names. Should they change depending on theme?

Reply:
I think the names might be best to stay the same as they're above quite saturated dynamic colours from the zone colour whether you're in dark or light mode.